scrape_apihub_opensearch.py

```diff
@@ -335,8 +366,18 @@ def get_existing_acqs(start_time, end_time, location=False):
 
     for item in hits:
         acq_ids.append(item.get("_source").get("metadata").get("id"))
+        missiondatatakeid = item['_source']['metadata'].get('missiondatatakeid')
+
+        if missiondatatakeid:
+            acq_id = item['_source']['metadata'].get('id')
+            ingestion_date = item['_source']['metadata'].get('ingestiondate')
+            mission_data_id_store[missiondatatakeid] = {
+                'id': acq_id,
```

Comment:
Looking through the usage of the mission_data_id_store, I don't see the "id" field being utilized anywhere. @DustinKLo Could you verify that? If it's not being used then please delete it from the dict.

Reply:
<img width="796" alt="Screen Shot 2019-06-05 at 9 39 19 AM" src="https://user-images.githubusercontent.com/13371881/58973891-e5edc880-8775-11e9-9881-72306a07e96f.png">

@NamrataM yes, we save it in the `metadata` field, i use it so i can remove `id`'s from `prods_missing` if scihub's acquisition is older than ours

Reply:
@DustinKLo Okay great, thanks for that. I need to revisit the logic of duplicate detection then. May need to add a few more constraints.